src/main/java/network/brightspots/rcv/CastVoteRecord.java

```diff
@@ -90,7 +89,7 @@
     this.precinct = precinct;
     this.precinctPortion = precinctPortion;
     this.ballotTypeId = ballotTypeId;
-    this.fullCvrData = new ArrayList<>();
+    this.fullCvrData = null;
```

Comment:
You can just delete this line (and the one for `computedId`).

Also, please add a comment explaining why this constructor exists/how it differs from the one below.

Reply:
We can't delete those lines for class variables which are static final - they need an explicit assignment to achieve their final destiny.  Comment is def good idea.